Unit testing with MockProvider causes an error stating Cannot find module 'lodash.isequal' from 'test-utils.js'

Unit testing using MockProvider as shown in https://www.apollographql.com/docs/guides/testing-react-components.html#MockedProvider fails. 

The following is the error 


    Cannot find module 'lodash.isequal' from 'test-utils.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
      at components/SiteHeaderWithData/node_modules/react-apollo/test-utils.js:881:19
      at Object.<anonymous>.ApolloClient__default (components/SiteHeaderWithData/node_modules/react-apollo/test-utils.js:2:68)
      at Object.<anonymous> (components/SiteHeaderWithData/node_modules/react-apollo/test-utils.js:5:2)


Currently`lodash.flowright` is added as a dependency. I think `loadash.isequal` should also be added

`lodash` was removed from dependency and only `lodash.flowright` is added as part of the following commit https://github.com/apollographql/react-apollo/commit/4eebfdc0f1614583ed877583285551ed4c39ddba#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

**Version**

- apollo-client@2.4.2
- react-apollo@2.2.3

--- a/package.json
+++ b/package.json
@@ -123,7 +123,6 @@
     "jest-junit": "5.1.0",
     "jsdom": "12.0.0",
     "lodash.includes": "^4.3.0",
-    "lodash.isequal": "^4.5.0",
     "lodash.times": "^4.3.2",
     "preact": "8.3.1",
     "preact-compat": "3.18.4",
@@ -151,6 +150,7 @@
     "hoist-non-react-statics": "^3.0.0",
     "invariant": "^2.2.2",
     "lodash.flowright": "^3.5.0",
+    "lodash.isequal": "^4.5.0",
     "prop-types": "^15.6.0"
   }
 }

